index.html

```diff
@@ -56,14 +56,20 @@ <h1>Welcome to Matthew Burgan's CSD120 Website</h1>
         </p>
         <table>
           <caption> Table 1 Current Courses</caption>
-          <tr><th rowspan="2">Term</th></tr><tr><th colspan="3">Course</th></tr>
-          <tr><th rowspan="6">Fall</th></tr>
-          <tr><th> Code </th><th>Name</th><th> hrs/week </th></tr>
-          <tr><th> CSD105</th><th>Python</th><th>3</th></tr>
-          <tr><th>CSD120</th><th>Intro to Web Development</th><th>4</th></tr>
-          <tr><th>MTH122</th><th>Computer Math</th><th>3</th></tr>
-          <tr><th>CS0104</th><th> Intro OS/LAN Admin</th><th>4</tr>
-          <tr><th colspan="3">Total Hours </th><td>14</td></tr> 
+          <thead>
+            <tr><th rowspan="2" scope="col">Term</th><th colspan="3" scope="colgroup">Course</th></tr>
+            <tr><th scope="col"> Code </th><th scope="col">Name</th><th scope="col"> hrs/week </th></tr>
+          </thead>
+  
+          <tbody>
+            <tr><th rowspan="4" scope="rowgroup">Fall</th><td> CSD105</td><td>Python</td><td>3</td></tr>
+            <tr><td>CSD120</td><td>Intro to Web Development</td><td>4</td></tr>
+            <tr><td>MTH122</td><td>Computer Math</td><td>3</td></tr>
+            <tr><td>CS0104</td><td> Intro OS/LAN Admin</td><td>4</tr>
+          </tbody>
+          <tfoot>
+            <tr><th colspan="3" scope="row">Total Hours </th><td>14</td></tr>
+          </tfoot> 
```

Comment:
Missing `<tbody>`, `<thead>` and `<tfoot>`
Your header cell were a bit mixed up
Lots of missing scopes